nix/preset/bash.nix

```diff
@@ -20,7 +20,7 @@ in {
       HOME = lib.mkDefault "/local/home";
       NIX_SSL_CERT_FILE = lib.mkDefault "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
       SSL_CERT_FILE = lib.mkDefault "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
-      # PATH = lib.makeBinPath config.dependencies;
+      PATH = lib.makeBinPath config.dependencies;
```

Comment:
```suggestion
```

Setting `PATH` from `config.dependencies` is handled by the [writers](https://github.com/input-output-hk/tullia/blob/565de3f76098aa0a3f13a4ef8c11f43eabdb196d/nix/writer/shell.nix#L23) through [`command.*.runtimeInputs`](https://github.com/input-output-hk/tullia/blob/565de3f76098aa0a3f13a4ef8c11f43eabdb196d/nix/module.nix#L139).

Hence your [original change](https://github.com/input-output-hk/tullia/pull/9#discussion_r1001976646) (which now seems to be force-pushed out) was correct.